### ISSUE TITLE
"Data Governance" Entitlement

### ISSUE THREAD

**Issue by jpalarchio** [MicrosoftDocs/OfficeDocs-O365ServiceDescriptions#68]
Something doesn't make sense in this documentation.  With the footnote are we saying that "Data Governance" is allowed for workloads other than Exchange but you need E3+ for Exchange?

When you look at the standalone plans, Exchange Online Plan 1 has a footnote stating you also need EOA yet Exchange Kiosk (the SKU with the least features) does not have that footnote.

Also, does "Manual retention/deletion policies" refer to the old MRM policies?  If so, that should be more clear.  And why would O365 F1 not include this entitlement yet Exchange Kiosk supposedly does?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 663a603e-482b-0a4f-55a9-229ceab11bb1
* Version Independent ID: 488b1daa-b459-c56d-11eb-59242af822b0
* Content: [Office 365 Security & Compliance Center](https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-securitycompliance-center)
* Content Source: [Office365-ServiceDescriptions/office-365-platform-service-description/office-365-securitycompliance-center.md](https://github.com/MicrosoftDocs/OfficeDocs-O365ServiceDescriptions-pr/blob/live/Office365-ServiceDescriptions/office-365-platform-service-description/office-365-securitycompliance-center.md)
* Service: **o365-administration**
* GitHub Login: @pebaum
* Microsoft Alias: **pebaum**

**Comment by ernelso** [comment]
Agreed...  This statement is contradictory and does not make sense:
1 Data governance enables users to create, publish, and manually apply labels to documents; import data using drive shipping or over the network; and create and apply retention/deletion policies across workloads, users, and groups. Retention policies applied to mailboxes require Office 365 Enterprise E3 or E5. 

Please remediate or clarify whether retention policies are available for E1?

**Comment by prabita** [comment]
@jpalarchio - Thanks for your feedback! I’ve reached out to the product team and will update this article as needed, pending their response.

**Comment by SMG999** [comment]
I would really like to know if **Advanced** Data Governance is included with Microsoft 365 Business (and Microsoft 365 E3 and Microsoft 365 E5) I can't find a clear up to date answer on whether it is. https://www.microsoft.com/en-us/microsoft-365/blog/2019/01/29/introducing-new-features-in-microsoft-365-to-help-prepare-for-the-next-wave-of-privacy-regulations/ seems to suggest so but it does not specify a SKU and previously Advanced Data Governance has been M365 E5-only so I am not sure if that has changed.

**Comment by EvgenyArtemyev** [comment]
There is weird discrepancy in the article about Data Governance.
the link in first table under Data Governance leads to https://docs.microsoft.com/office365/securitycompliance/**data-loss-prevention-policies** 
In comparison, the link in the second table under Data Governance leads to another page 
https://docs.microsoft.com/office365/securitycompliance/**retention-policies**

based on the description it should lead to another article 
https://docs.microsoft.com/en-us/office365/securitycompliance/**labels**

Additionally, the note is fairly unclear:
**Data governance** enables users to create, publish, and manually apply labels to documents; import data using drive shipping or over the network

Basically, the users can merely **apply** labels, they cannot create or publish
Only admins can create and publish the Labels.

After all, it's stated that users can publish labels (aka create manual retention policy). Essentially, publish means creating the Retention policy, therefore if they have entitlement to **Data Governance** for O365Business plans they also have entitlement for **manual retention policy** for the same Business plans as well. However we see NO in Business plans next to Manual Retention Policies

Please team, clarify all of these discrepancies.

**Comment by aaronlemieux** [comment]
This item has been updated.
